api/docker-compose.debug.yml

```diff
@@ -0,0 +1,10 @@
+version: '2.1'
+
+services:
+  api:
+    image: sustainable-mobility-api
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8080:8080
```

Comment:
Do we need this file? I think in their current form the two files are equivalent anyways.